clap_complete/src/dynamic/shells/zsh.rs

```diff
@@ -0,0 +1,60 @@
+/// Completion support for zsh
+#[derive(Copy, Clone, PartialEq, Eq, Debug)]
+pub struct Zsh;
```

Comment:
This should have the same level of tests as fish

https://github.com/clap-rs/clap/blob/5da658c5a06e369a5dc4c073f3f043cab05c403a/clap_complete/tests/testsuite/fish.rs#L168-L222